src/config.rs

```diff
@@ -368,6 +381,10 @@ impl fmt::Display for CommandNameAndArgs {
             CommandNameAndArgs::String(s) => write!(f, "{s}"),
             // TODO: format with shell escapes
             CommandNameAndArgs::Vec(a) => write!(f, "{}", a.0.join(" ")),
+            CommandNameAndArgs::Structured {
+                env: _,
+                command: cmd,
+            } => write!(f, "{}", cmd.0.join(" ")),
```

Comment:
nit: better to `write!(f, "{k}={v} ")` for each env. It'll be included in error message.

Reply:
Good idea. Will fix.